study-builder/tenants/cmi/change_password_email.html

```diff
@@ -0,0 +1,140 @@
+<html>
+<meta name="format-detection" content="telephone=no,date=no,address=no,email=no,url=no"/>
+<head>
+  <title>
+    {% if client.client_metadata.study == "ANGIO" %}
+      Angiosarcoma Project
+    {% elsif client.client_metadata.study == "cmi-brain" %}
+      Brain Cancer Project
+    {% elsif client.client_metadata.study == "cmi-mbc" %}
+      Metastatic Breast Cancer Project
+    {% elsif client.client_metadata.study == "CMI-OSTEO" %}
+      Osteosarcoma Project
+    {% endif %}
+    Password Reset
+  </title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700" rel="stylesheet">
+  <style type="text/css">
+    .ExternalClass,
+    .ExternalClass div,
+    .ExternalClass font,
+    .ExternalClass p,
+    .ExternalClass span,
+    .ExternalClass td,
+    img {
+      line-height: 100%;
+    }
+
+    #outlook a {
+      padding: 0;
+    }
+
+    .ExternalClass,
+    .ReadMsgBody {
+      width: 100%;
+    }
+
+    a,
+    blockquote,
+    body,
+    li,
+    p,
+    table,
+    td {
+      -webkit-text-size-adjust: 100%;
+      -ms-text-size-adjust: 100%;
+    }
+
+    table,
+    td {
+      mso-table-lspace: 0;
+      mso-table-rspace: 0;
+    }
+
+    img {
+      -ms-interpolation-mode: bicubic;
+      border: 0;
+      height: auto;
+      outline: 0;
+      text-decoration: none;
+    }
+
+    table {
+      border-collapse: collapse !important;
+    }
+
+    body {
+      height: 100%!important;
+      margin: 0;
+      padding: 0;
+      font-family: 'Roboto', sans-serif;
+    }
+
+    @font-face {
+      font-family: 'Roboto', sans-serif;
+      src: url(https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700);
+      font-weight: 300;
+      font-style: normal;
+    }
+  </style>
+</head>
+<body>
+
+{% assign from = "The Count Me In Team" %}
+{% assign study = client.client_metadata.study %}
+{% assign iconUrl = "https://storage.googleapis.com/dev-count-me-in/logo-count-me-in.png" %}
+{% assign studyLink = "https://www.joincountmein.org" %}
+
+{% if study == "ANGIO" %}
+  {% assign from = "The Angiosarcoma Project Team" %}
+  {% assign iconUrl = "https://storage.googleapis.com/cmi-study-dev-assets/angio/project-logo.png" %}
+  {% assign studyLink = "https://dev.ascproject.org" %}
+{% endif %}
+
+{% if study == "cmi-brain" %}
+  {% assign from = "The Brain Cancer Project Team" %}
+  {% assign iconUrl = "https://storage.googleapis.com/cmi-study-dev-assets/brain/project-logo.png" %}
+  {% assign studyLink = "https://dev.braincancerproject.org" %}
+{% endif %}
+
+{% if study == "cmi-mbc" %}
```

Comment:
@ssettipalli can you check the various MBC configs in this PR?

Reply:
looks good